### ISSUE TITLE
User dictionary list shows all languages - always

### ISSUE THREAD

**Issue by snomos** [divvun/giellakbd-ios#167]
On Android, the languages listed are only those with content.

On iOS it shows the full list of languages available in the app, not only the ones with content, or a list corresponding to the keyboards enabled.

Most of these languages will be irrelevant to the user, I suggest the list should be limited to the keyboards enabled by the user.

**Comment by snomos** [comment]
Correction: On Android, the languages listed are those that have been used at least once. That is, also languages with no content are listed when they have been used. The list does not contain all languages/keyboards enabled.

This looks to me as a good solution, and should be adopted on iOS. Alternatively it could be all and only the languages for the enabled keyboards.

**Comment by snomos** [comment]
Example screenshot:

![image](https://user-images.githubusercontent.com/1255285/95285891-8e300780-086a-11eb-8cba-c8108291c63f.jpeg)

The three marked languages are not enabled, have never been, and should not be listed. 

I suggest three groups of languages:

- Enabled
- Disabled but with user dictionary content (somehow distinct from the first group)
- Disabled and no user dictionary content (never showed)

It should be the same on both Android and iOS.

**Comment by dylanhand** [comment]
The list now only shows locales that are either enabled, or have content in their user dictionary (or both). Disabled keyboards with no dictionary content are hidden.
